src/inspect_ai/_view/www/src/index.ts

```diff
@@ -0,0 +1,46 @@
+// CSS Styles - Import this to get all required styles
```

Comment:
General note: Can you review the below exports and focus on trying to create a small footprint that enables the most basic hosting of the App (rather than a comprehensive API)? Some of the types and APIs aren't really ready to be used externally so I'd rather not export them unless absolutely necessary (in which case we may to do a bit more cleanup).